ui/src/app/help/components/help.tsx

```diff
@@ -23,14 +23,6 @@ export const Help = () => (
                             <a href={`download/argocd-linux-amd64`} className='user-info-panel-buttons argo-button argo-button--base'>
                                 <i className='fab fa-linux ' /> Linux
                             </a>
-                            &nbsp;
-                            <a href={`download/argocd-darwin-amd64`} className='user-info-panel-buttons argo-button argo-button--base'>
```

Comment:
The `argocd-darwin-amd64` and `argocd-windows-amd64.exe` and mentioned in https://github.com/argoproj/argo-cd/blob/09c5d5f42abe2a8a7a36946cd69c15c625ec98ae/server/server.go#L811

Please remove it as well.

Reply:
Also `server.go` should register `argocd-linux-amd64` or `argocd-linux-arm` based on architecture type. I think we can inject architecture using `LDFLAGS` here: https://github.com/argoproj/argo-cd/blob/09c5d5f42abe2a8a7a36946cd69c15c625ec98ae/Makefile#L131

Reply:
Good call. I used  `runtime.GOARCH` instead of `LDFLAGS` since it's simpler.